packages/node_modules/samples/browser-call-with-screenshare/app.js

```diff
@@ -330,8 +328,7 @@ function bindMeetingEvents(meeting) {
       document.getElementById('remote-view-audio').srcObject = media.stream;
     }
     if (media.type === 'remoteShare') {
-      // Remote share streams become active immediately on join, even if nothing is being shared
-      remoteShareStream = media.stream;
+      document.getElementById('remote-screen').srcObject = media.stream;
```

Comment:
condition to set the streams , check meeting.shareShare === 'remotet_share"

Reply:
isnt this already occuring in line 332: `if (media.type === 'remoteShare')` ?

Reply:
I also noticed that while stepping, `meeting:startedSharingRemote` is called before the remoteShareStream is set, so remoteShareStream is always undefined.